assets/src/components/withCharacterCounter/withCharacterCounter.js

```diff
@@ -52,7 +52,7 @@ const withCharacterCounter = ( WrappedComponent ) => {
     }
 
     render() {
-      const { characterLimit, warningThreshold, onChange, ...passThroughProps } = this.props;
+      const {characterLimit, ...passThroughProps} = this.props;
```

Comment:
I think that here again we need to keep excluding these 2 props from `passThroughProps`, since we override these functionalities